src/Rest.js

```diff
@@ -16,14 +16,25 @@ define([
 		__PutDirectives = declare(Store.PutDirectives, __HeaderOptions),
 	=====*/
 
-	return declare(Request, {
+	return declare(Request.default, {
 
 		// stringify: Function
 		//		This function performs the serialization of the data for requests to the server. This
 		//		defaults to JSON, but other formats can be serialized by providing an alternate
 		//		stringify function. If you do want to use an alternate format, you will probably
 		//		want to use an alternate parse function for the parsing of data as well.
-		stringify: JSON.stringify,
+		stringify: null,
+
+		constructor: function() {
+			// TODO - this is being initialized here because due to the way TS initializes variable the default on the
+			// prototype would be overridden
+			this.stringify = JSON.stringify;
```

Comment:
Given that you now implemented `_initialize`, would it make more sense to just move this there?  (Maybe we don't need the TODO after all.)